extensions/amp-story/1.0/test/validator-amp-story-grid-layer-error.out

```diff
@@ -0,0 +1,43 @@
+FAIL
+|  <!--
+|    Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+|    Licensed under the Apache License, Version 2.0 (the "License");
+|    you may not use this file except in compliance with the License.
+|    You may obtain a copy of the License at
+|        http://www.apache.org/licenses/LICENSE-2.0
+|    Unless required by applicable law or agreed to in writing, software
+|    distributed under the License is distributed on an "AS-IS" BASIS,
+|    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+|    See the License for the specific language governing permissions and
+|    limitations under the license.
+|  -->
+|  <!--
+|    Test Description:
+|    Tests for the live story functionality inside amp-story.
+|  -->
+|  <!doctype html>
+|  <html amp lang="en">
+|    <head>
+|      <meta charset="utf-8">
+|      <script async src="https://cdn.ampproject.org/v0.js"></script>
+|      <script async custom-element="amp-story"
+|          src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+|      <title>amp-story-grid-layer</title>
+|      <meta name="viewport"
+|            content="width=device-width,minimum-scale=1,initial-scale=1">
+|      <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+|      <link rel="canonical" href="index.html">
+|    </head>
+|
+|    <body>
+|      <amp-story standalone>
+|        <amp-story-page>
+|          <amp-story-grid-layer aspect-ratio="a:b" template="vertical">
+>>     ^~~~~~~~~
+amp-story/1.0/test/validator-amp-story-grid-layer-error.html:35:6 The attribute 'aspec-ratio' in tag 'amp-story-grid-layer' is set to the invalid value 'a:b'. (see https://amp.dev/documentation/components/amp-story)
```

Comment:
this is the correct error, but unsure why it says `aspec-ratio` instead of `aspect-ratio`.

Reply:
That's just me testing it.